buildSrc/src/main/kotlin/org/jetbrains/exposed/gradle/Versions.kt

```diff
@@ -1,30 +1,27 @@
 package org.jetbrains.exposed.gradle
 
 object Versions {
-    const val kotlin = "1.7.21"
-    const val kotlinCoroutines = "1.6.4"
+    const val kotlin = "1.9.0"
```

Comment:
Looks like this is not being used anywhere

Reply:
thanks, fixed